3rdParty/Storm/Source/storm.h

```diff
@@ -825,8 +825,10 @@ SMemReAlloc(
 #define SLOG_FILE         -4
 #define SLOG_EXCEPTION    -5
 
-void GetBasePath(char *buffer, size_t size);
-void GetPrefPath(char *buffer, size_t size);
+
+// void GetBasePath(char *buffer, size_t size);

```

Comment:
Delete thease they where added by me and don't really belong in this file

Reply:
Done